Developer

```diff
@@ -0,0 +1,90 @@
+#######################################
+# Developer file for host development machine
+#######################################
```

Comment:
```suggestion
###############################################
# Developer file for host development machine #
###############################################
```

😄

Reply:
Honestly it is much better :)

Reply:
We can either unify it like that in the new files or just simplify it all and leave a single `######` line under the text like Markdown.

Reply:
This suggestion still stands, can we just simplify the sections so we don't bother in the future?